Clinical-Genomics/demultiplexing
# Add padding of d8 indexes in 'demux sheet fetch' if no d10 indexes exist.

This PR adds padding of d8 indexes to `demux sheet fetch` for cases where there's no d10 indexes present in LIMS. Previously the sample sheet creation would abort with the message `Padding is only allowed in combination with dual 10 indexes!`

#### How to test:

1. Install on stage.
2. Make sure 'demux' points to production LIMS.
3. Run `demux sheet fetch --application nova --longest --pad HJMJJDSXX`
4. Check the created sample sheet.

#### Expected outcome:

1. All indexes should be of length 10. 
2. All indexes called `index1` should end in `AT`.
3. All indexes called `index2` should end in `AC`.

### Reviewed:

- [x] code approved by @ingkebil 
- [x] test approved by @ingkebil
